devel/libcbor/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libcbor
+license             MIT License
+version             0.8.0
+revision            0
+categories          devel
+platforms           darwin
+maintainers         @trodemaster \
+                    openmaintainer
+description         library for parsing and generating CBOR
+PortGroup           cmake 1.1
+
+long_description    libcbor is a C library for parsing and generating CBOR, the general-purpose schema-less binary data format.
+homepage            https://github.com/PJK/libcbor
+master_sites        https://github.com/PJK/libcbor/archive/
+distfiles           v${version}.tar.gz
+checksums               rmd160  ddd307e64b50f2a777a88e89e10e3891571226a2 \
+                        sha256  618097166ea4a54499646998ccaa949a5816e6a665cf1d6df383690895217c8b \
+                        size    267044
+configure.args      --mandir=${prefix}/share/man
+
+depends_lib        port:cmake
```

Comment:
not needed -- automatically added by the cmake 1.1 PortGroup

Reply:
(and it would be a depends_build if you did need it, as it's only needed at build time).